src/utils.ts

```diff
@@ -92,7 +92,9 @@ export function todayInMinutes() {
   return today.diff(dayjs().startOf('day'), 'minute')
 }
 
-export function modeToNum(mode: Mode, current?: dayjs.Dayjs | Date): number {
+export function modeToNum(mode: Mode, current?: dayjs.Dayjs | Date, direction?: String): number {
```

Comment:
use `HorizontalDirection` interface.

```suggestion
export function modeToNum(mode: Mode, current?: dayjs.Dayjs | Date, direction?: HorizontalDirection): number {
```